packages/core/src/extensions/Blocks/nodes/Block.module.css

```diff
@@ -31,6 +31,8 @@ NESTED BLOCKS
   position: relative;
 }
 
+/* TODO: Should be different for light/dark editor class, but can't check since
```

Comment:
Maybe we can:
- use a regular "dark" class on the editor? (instead of via modules)
- set data-theme="dark" on the editor?